extractpython.py

```diff
@@ -0,0 +1,35 @@
+import sys, re, os, json
+from grammars.grammars import python
+from extractor import find_subtree, remove_tabs_newlines
+
+def expr_filter(n):
+    return n.symbol.name == "testlist" and n.left_sibling()
+
+if __name__ == "__main__":
+    output = sys.argv[1]
+    results = []
+    folder = "../../../flowblade/flowblade-trunk/Flowblade/"
+    for filename in os.listdir(folder):
+        if not filename.endswith(".py"):
```

Comment:
`os.path.splitext`?

Reply:
Isn't that the same thing? I would even argue that `splitext` is less efficient as it allocates memory.

Reply:
`splitext` is more portable. Efficiency isn't a concern here :)

Reply:
Out of curiosity: How is it more portable?

Reply:
Not every platform uses "." as a separator.

Reply:
I see. Fair enoug.